src/Echidna.Tests/Mapping/DictionaryMappingTest.cs

```diff
@@ -6,7 +6,7 @@ namespace Medallion.Data.Tests.Mapping;
 internal class DictionaryMappingTest
 {
     [Test]
-    public void TestCanMapToIReadOnlyDictionary([Values] Db db)
```

Comment:
Couldn't use all `Values` because `MariaDb` and `Oracle` are not set up in the CI pipeline.